build.yaml

```diff
@@ -663,29 +764,45 @@ common_cli:
     pip install virtualenv
     brew install libffi
     export PKG_CONFIG_PATH=/usr/local/Cellar/libffi/3.0.13/lib/pkgconfig/; pip install bcrypt
+    # brew test is failling unless we install these packages
+    pip install docutils
+    pip install botocore
+    pip install boto
 
   package_jenkins_osx_yosemite_box: &package_jenkins_osx_yosemite_box |
     # we now proceed to package our vagrant box
-    vagrant halt
-    vagrant package --output jenkins-osx-yosemite.box
-    JOB_EXIT_STATUS="\$( updateExitStatus \$? )"
-    vagrant box add --force 'jenkins-osx-yosemite'   jenkins-osx-yosemite.box
-    JOB_EXIT_STATUS="\$( updateExitStatus \$? )"
-    rm -f jenkins-osx-yosemite.box
-    vagrant destroy -f
-    JOB_EXIT_STATUS="\$( updateExitStatus \$? )"
-    exit \${JOB_EXIT_STATUS}
+    # let's stop it and wait a bit, as sometimes the box is not fully down
+    # when the vagrant halt command finishes.
+    vagrant halt ; sleep 60
```

Comment:
:question: Sleeping is a bit of a compromise. Either it's going to be too long a sleep and create unnecessary delays (perhaps irrelevant given how slow everything else is), or it's going to be too short and we'll get bugs. Is there a way to poll until the box is fully down?